face/capture/src/main/java/com/simprints/face/capture/usecases/GetAndTurnOffPreparationInstructionsShowingUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.simprints.face.capture.usecases
+
+import androidx.annotation.VisibleForTesting
+import com.simprints.infra.security.SecurityManager
+import javax.inject.Inject
+import javax.inject.Singleton
+import androidx.core.content.edit
+
+@Singleton
+class GetAndTurnOffPreparationInstructionsShowingUseCase @Inject constructor(
+    private val securityManager: SecurityManager,
+) {
+    operator fun invoke(): Boolean {
+        val sharedPrefs = securityManager.buildEncryptedSharedPreferences(FILENAME_FOR_INSTRUCTIONS_SHOWING_SHARED_PREFS)
+        val areInstructionsShowing = sharedPrefs.getBoolean(INSTRUCTIONS_SHOWING_PREFERENCE_KEY, true)
```

Comment:
It might be a bit of a personal preference, but I would reverse this flag into "instructionsShown" and set it to "true" on first call which is a bit simpler behaviour/logic.